src/content/docs/it/concepts/islands.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Isole Astro
+description: "Le Isole Astro (noto anche come Isole Component) sono un modello di architettura web sviluppato da Astro. Il concetto di “architettura delle isole” è stato coniato per la prima volta dalla sviluppatrice frontend di Etsy, Katie Sylor-Miller, nel 2019, ed è stato ampliato dal creatore di Preact, Jason Miller."
```

Comment:
- I would use "note anche..." (le isole), on line 8 too.
- And "Isole di componenti" (like you wrote on line 8) seems clearer. In general, I noticed the description in the frontmatter (i.e. this section) is the same of the first section of the article, so maybe it's a good idea to use the exact same translation.

Reply:
I applied the fixes!

Reply:
I think you forgot to replace "noto" with "note", since "isole" is feminine plural

Reply:
I applied the fixes!